### DISCUSSION TITLE
Question about firebase init

### DISCUSSION THREAD

**Discussion by alexookah** [capawesome-team/capacitor-firebase#615] [General]
Hello,
I recently upgraded from capacitor 4 to 6 and also angular 17 and ionic.

For some reason in android update users had to re-login but not in iOS.
I was wondering if this is the correct way to init my firebase app. auth, storage & messaging.
(using the same code for web, pwa, capacitor android & capacitor iOS)

![Screenshot 2024-04-19 at 19 17 21](https://github.com/capawesome-team/capacitor-firebase/assets/9624843/a6741e5b-ec6a-4f24-99f6-8bfd1dccee03)

**Comment by robingenz** [comment]
I don't use Angularfire but this looks good to me.

**Reply by obermoser** [reply]
hey there. looks good to me, but when you initialize auth, just remove the persistence part as it is not needed when you use FirebaseAuthentication from this repo.

**Reply by alexookah** [reply]
@obermoser why persistance is not needed? I think it's needed for offline. or?
